sdk/automation/azure-mgmt-automation/tests/recordings/test_cli_mgmt_automation.test_automation.yaml

```diff
@@ -165,7 +164,7 @@ interactions:
       x-content-type-options:
       - nosniff
       x-ms-location:
-      - https://management.azure.com/subscriptions/92f95d8f-3c67-4124-91c7-8cf07cdbf241/resourceGroups/test_cli_mgmt_automation_test_automation53db109a/providers/Microsoft.Automation/automationAccounts/myAutomationAccount9/modules/module1?api-version=2019-06-01
+      - https://management.azure.com/subscriptions/92f95d8f-3c67-4124-91c7-8cf07cdbf241/resourceGroups/rgpy-test_cli_mgmt_automation_test_automation53db109a/providers/Microsoft.Automation/automationAccounts/myAutomationAccount9/modules/module1?api-version=2019-06-01
```

Comment:
Is this a subscription id in the `x-ms-location` header too?

Reply:
Yeah, it is. But this one is in header and it's been around here for a long time . This PR is used to solve secrets in requests and responses.
We need another PR to fix `replace_header` to solve it.